### DISCUSSION TITLE
How do I transpose the axes of a 2D plot?

### DISCUSSION THREAD

**Discussion by nsmith-** [scikit-hep/hist#133] [Q&A] [answered]
I'd like to plot a 2D hist with the x and y axes swapped, can there be a `h.T` property or something like it? Also would be nice to have a general axis permutation tool.

**Comment by henryiii** [comment, marked as answer]
`h.project(1,0)`, perhaps is what you are looking for?

**Reply by nsmith-** [reply]
Nice! Didn't think of that.
